CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+### v4.1.2 (2017-01-12)
+
+We have a twee little release this week as we come back from the holidays.
+
+#### 0.12 IS UNSUPPORTED NOW (really)
+
+After earlier jumping the gun a little, we can now officially remove 0.12
+from our supported versions list.  This means we no longer run the test
+suite under 0.12 and if you run npm under 0.12 you'll receive a warning.
+
+* [`c7bbba8`](https://github.com/npm/npm/commit/c7bbba8744b62448103a1510c65d9751288abb5d)
+  Remove 0.12 from our supported versions list.
+  ([@iarna](https://github.com/iarna))
+
+#### WRITING TO SYMLINKED FILES
+
```

Comment:
might be a neat place to remind people that there is a `npm link`, and link to https://docs.npmjs.com/cli/link